content/developers/dogstatsd/datagram_shell.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Datagram Format and Shell Usage
+kind: documentation
+description: Overview of the datagram format used by DogStatsD as well as (advanced) shell usage.
+further_reading:
+- link: "developers/dogstatsd"
+  tag: "Documentation"
+  text: "Introduction to DogStatsD"
+- link: "developers/libraries"
+  tag: "Documentation"
+  text: "Official and Community-contributed API and DogStatsD client libraries"
+- link: "https://github.com/DataDog/dd-agent/blob/master/dogstatsd.py"
+  tag: "Github"
+  text: "DogStatsD source code"
+---
+
+
+This section specifies the raw datagram format for each data type that DogStatsD accepts. This isn't required reading if you're using any of the DogStatsD client libraries; however, if you want to write your own library, or use the shell to send metrics or events, then read on.
+
+## Datagram Format
+
+### Metrics
+
+`metric.name:value|type|@sample_rate|#tag1:value,tag2`
+
+- `metric.name` — a string with no colons, bars, or @ characters. See the [metric naming policy][1].
```

Comment:
Would a table layout be more readable here?

Reply:
I don't have a strong opinion either way. Imho we leave the list for now and switch it up later if we like.